src/multidim/arithmetic.jl

```diff
@@ -0,0 +1,18 @@
+# This file is part of the IntervalArithmetic.jl package; MIT licensed
+
++(a::IntervalBox, b::IntervalBox) = IntervalBox( a.v .+ b.v )
++(a::IntervalBox, b::Real) = IntervalBox( a.v .+ b )
++(a::Real, b::IntervalBox) = IntervalBox( a .+ b.v )
+
+-(a::IntervalBox, b::IntervalBox) = IntervalBox( a.v .- b.v )
+-(a::IntervalBox, b::Real) = IntervalBox( a.v .- b )
+-(a::Real, b::IntervalBox) = IntervalBox( a .- b.v )
+-(a::IntervalBox) = IntervalBox( .- a.v )
+
+*(a::IntervalBox, b::Real) = IntervalBox( a.v .* b )
+*(a::Real, b::IntervalBox) = IntervalBox( a .* b.v )
+
+/(a::IntervalBox, b::Real) = IntervalBox( a.v ./ b )
+
+Base.broadcast(f, X::IntervalBox) = IntervalBox(f.(X.v))
+Base.broadcast(f, X::IntervalBox, Y::IntervalBox) = IntervalBox( f.(X.v, Y.v) )
```

Comment:
Neat!

Reply:
While I truly think that this broadcasting change is very nice, I just noticed that, in order to use a function (say `exp`) on an `IntervalBox`, you *do* require the dot-notation. The following uses the current commit (93bb8e0f4) of this PR:
```julia
julia> A = IntervalBox(1..2, 3..4)
[1, 2] × [3, 4]

julia> exp.(A) # this is implemented here
[2.71828, 7.38906] × [20.0855, 54.5982]

julia> exp(A) # throwing an error here is uncomfortable!
ERROR: MethodError: no method matching exp(::IntervalArithmetic.IntervalBox{2,Float64})
Closest candidates are:
  exp(::Float16) at math.jl:950
  exp(::Complex{Float16}) at math.jl:951
  exp(::BigFloat) at mpfr.jl:517
  ...
```

This is sort of uncomfortable, because you force the user to write whatever functions using the dot notation instead of the usual functions. This is actually not a problem, a priori, but then it should be explicitly written somewhere. My idea with #151 was to avoid this, i.e., use directly `exp(A)`.

Should we allow `exp(A)` to simply return `exp.(A)`, or shall we leave it as it is now?

Reply:
Julia has moved to requiring `exp.(v)` to make explicit that you want to apply `exp` element-wise, since `exp` of a vector does not make sense.

I don't understand the context where `exp` of an `IntervalBox` makes sense. I don't actually understand your `Taylor` example.

Reply:
> Julia has moved to requiring exp.(v) to make explicit that you want to apply exp element-wise, since exp of a vector does not make sense.

Fair enough.